src/core/linear_solver/src/preconditioner/4C_linear_solver_preconditioner_muelu.hpp

```diff
@@ -31,10 +31,29 @@ namespace Core::LinearSolver
    * A. Prokopenko, C. M. Siefert, R. S. Tuminaro, T. A. Wiesner:
    * MueLu User's Guide, Technical Report, Sandia National Laboratories, SAND2019-0537, 2019
    */
-  class MueLuPreconditioner : public PreconditionerTypeBase
-  {
-   public:
-    MueLuPreconditioner(Teuchos::ParameterList& muelulist);
+  class MueLuPreconditioner : public PreconditionerTypeBase {
+  public:
+    MueLuPreconditioner(Teuchos::ParameterList &muelulist);
+
+    static void printParameterList(const Teuchos::ParameterList &pL, int indentLevel = 0) {
```

Comment:
I guess this function is left from debugging and you can remove it?

Reply:
Thanks for your tips, @sebproell. I ran the `set_up_dev_env.sh script`, so I think the formatting should be fixed now. And the `printParameterList` function was indeed accidentally left from debugging, so I removed it. Thank you!